packages/components/pie-toast/src/index.ts

```diff
@@ -119,6 +218,14 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
     protected async updated (_changedProperties: PropertyValues<this>) {
         if (_changedProperties.has('isOpen') && this.isOpen) {
             dispatchCustomEvent(this, ON_TOAST_OPEN_EVENT, { targetNotification: this });
+
+            if (this.duration !== null) {
+                this.createAutoDismissEventListeners();
+            }
+        }
+
+        if (_changedProperties.has('isOpen') && !this.isOpen && this._abortController) {
```

Comment:
The method already checks for the abort controller so we don't need to check it here too

Reply:
Agreed